msal4j-sdk/pom.xml

```diff
@@ -240,6 +240,8 @@
                 <configuration>
                     <source>8</source>
                     <target>8</target>
+                    <testSource>11</testSource>
```

Comment:
Seems like the builds are failing due this telling Maven to use Java 11 when building the files for the tests. 

Is that testng update that needed Java 11 part of this release?